cdc_iterator_integration_test.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 func testCdcIterator(ctx context.Context, t *testing.T, is *is.I) (common.Iterator, func()) {
-	db := testutils.Connection(t)
+	db := testutils.Connection(t).Conn()
```

Comment:
@alarbada I feel like. `Connection()` no longer returns the actual db connection and I find these chaining methods misleading and not very go idiomatic. What happened with the other existing calls to `Connection()` that's not using the now returned `Conn()` ?

Reply:
Yeah I don't like it either, but snapshot and cdc iterator integration tests are using a custom sqlx connection. Maybe the